our-pricing/index.html

```diff
@@ -0,0 +1,371 @@
+                    <div class="table-data">
+                      <p>Out-Stn Driver allowance Per Day/Trip: </p>
+                      <p> 500/-</p>
+                    </div>
+                    <div class="table-data">
+                      <p>Out-Stn night halt Charge Per Night: </p>
+                      <p> 400/-</p>
+                    </div>
+                  </div>
+                </div>
+              </div>
+              
+                <div class="desc-wrap">
+                    <p class="disclosure">*Disclosure / Terms and Conditions</p>
+                    <ol >
+                        <li>Time & Km will be calculated from garage to garage. (Maximum - 20km & 90 minutes up & down will be added).
+                           
+                           
+                           
+                           
+                           </li>
+                            <li> Should be fill the reporting time, reporting Km, Releasing Time & Km by the PASSENGER at the time of use.</li>
+                            <li> A night allowance will be charged for the driver Rs-200/- before 06:00 & after 21:00 if the vehicle is used more than 10 hours.</li>
+                            <li> Toll, Park & other RTO charges if any will be charged extra.</li>
+                            <li> GST will be charged as per the government rules and regulations.</li>
+                            <li> Billing date is 1st & 15th day of every month and Payment should be released within 15 days after receiving invoices.</li>
+                    </ol>
+                    <p class="disclosure">"Currently, we offer our services in Ahmedabad, Agra, Bangalore, Baroda, Bhopal, Bhubaneshwar, Chandigarh, Chennai, Cochin, Delhi, Goa, Gurgaon, Hyderabad, Indore, Jaipur, Kolkata, Mumbai, Noida, Pune, and Vizag. We are working to extend support to more cities."</p>
+                </div>
+        
+            </div>
+          </section>
+      </div>
+    </main>
+  </div>
+  <footer>
+    <div class="container">
+      <div class="left-wrap">
+        <div class="img-wrap">
+          <a href="https://alphytoursandtravels.com/"><img loading="lazy" src="./../assets/images/alphy_logo.png"
+              alt="Logo for Alphy tours and travels" width="847" height="294"></a>
+        </div>
+      </div>
+      <div class="right-wrap">
+        <ul>
+          <li><a href="/privacy-policy/">Privacy Policy</a></li>
+          <li><a href="mailto:alphytoursandtravels@gmail.com">Email : alphytoursandtravels@gmail.com</a></li>
+          <li><a href="tel:+91 95949439405">Phone: +91 95949439405</a></li>
+          <li><a href="tel:+91 7710087056">Enquiry & Booking : +91 7710087056</a> / <a href="tel:+91 7710087057">+91 7710087057</a></li>
+        </ul>
+      </div>
+    </div>
+    <p class="copyright">Copyright © 2024 | Alphy tours and travels</p>
+  </footer>
+
+  <script>
+
+    document.addEventListener('DOMContentLoaded', function () {
```

Comment:
Create a seperate js yo use it on home and out pricing

Reply:
Created a Separate file for accordion js